libraries/AP_HAL_ESP32/targets/esp32/esp-idf/sdkconfig

```diff
@@ -741,10 +741,10 @@ CONFIG_LWIP_CHECKSUM_CHECK_ICMP=y
 # end of Checksums
 
 CONFIG_LWIP_TCPIP_TASK_STACK_SIZE=3072
-CONFIG_LWIP_TCPIP_TASK_AFFINITY_NO_AFFINITY=y
-# CONFIG_LWIP_TCPIP_TASK_AFFINITY_CPU0 is not set
+# CONFIG_LWIP_TCPIP_TASK_AFFINITY_NO_AFFINITY is not set
+CONFIG_LWIP_TCPIP_TASK_AFFINITY_CPU0=y
```

Comment:
These TASK_AFFINITY_CPU0 are "pinning" the internal "non-open" wifi functionalinty to cpu0 - is this actually resulting in better performance , as its putting them on the same core as the above 'xTaskCreatePinnedToCore' calls, wihich are also on core 0?